src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -111,7 +111,6 @@ export class LocalGitRepoHelper {
      */
     public async addContentToFile(content: string, pathToFile: string): Promise<string> {
         fs.writeFileSync(pathToFile, content);
-        await vscode.workspace.saveAll(true);
```

Comment:
THis is done for all types of configurer. which i think is valid, we should not saved any unsaved files and writeFileSync as it is writing so it will save the file as well

Reply:
@bishal-pdMSFT is this the required behaviour?

Reply:
IMO in remote configurer, from a functionality point of view, we do not need files to be saved locally as we just need the content. There is no use of locally saved file. Infact that is a problem as we might to remove those files once remote provisioning has completed otherwise user will see those files as checked out.

Reply:
And I think the bug is also whether we should save just the workflow/manifest files rather than all files in workspace

Reply:
@bishal-pdMSFT , For this https://github.com/github/azure-integration/issues/72 bug we have removed the code to save  any unsaved files.
For the scenario where `git status` will show workflow files in untracked status, the only solution is i think we delete those workflow files from user workspace. The other approach like showing the files to users by showing only buffer with some vscode api, for this as well we need to create directory (.github or manifest, if this is first time) and adding adding empty files. In that case as well `git status` will show them as untracked files, even though they will be empty.

But other thing is , if we delete the workflow files then this is will be inconsistent with how our extension handles workflow files before remote configurer change,

Reply:
@tiwarishub lets merge without changing the existing behavior. This behavior is already shipped and hence not a blocker for the PR. This will unblock us to ship node changes.
We can take up this discussion separately and based on the conclusion create a separate PR.
cc @anuragc617

Reply:
sure @bishal-pdMSFT